website/routes.py

```diff
@@ -882,6 +882,24 @@ def make_url_map(app):
             project_views.register.node_registration_embargo_disapprove,
             OsfWebRenderer('error.mako', trust=False)
         ),
+        Rule(
```

Comment:
Because we guarantee all URL paths from the OSF, it would probably be better to put the approval portion into a query argument. something like `/project/{pid}/registration?approval=<token>`. Same with the disapproval below. Also, if the PID is of the registration draft rather than the main project, then I'd skip the /registration at the end and just do the approval. If not, how do you know which registration is being approved?